framework/integration/client/config_map.go

```diff
@@ -1,22 +1,22 @@
 // +build k8srequired
 
-package integration
+package client
 
 import (
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-func createConfigMap(configMap *corev1.ConfigMap) (*corev1.ConfigMap, error) {
-	createConfigMap, err := k8sClient.CoreV1().ConfigMaps(namespace).Create(configMap)
+func CreateConfigMap(configMap *corev1.ConfigMap) (*corev1.ConfigMap, error) {
+	createConfigMap, err := k8sClient.CoreV1().ConfigMaps(configMap.Namespace).Create(configMap)
 	if err != nil {
 		return nil, err
 	}
 
 	return createConfigMap, nil
 }
 
-func getConfigMap(name string) (*corev1.ConfigMap, error) {
+func GetConfigMap(name, namespace string) (*corev1.ConfigMap, error) {
```

Comment:
FYI In resource URL and client-go namespace goes before name. IMO it's more natural.

Reply:
I am not sure about this, I find alphabetical more consistent here. Going to leave it like this for now.